test_unstructured_ingest/test-ingest.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154
+
+set -u -o pipefail
```

Comment:
```suggestion
# Make sure shell exits when there are empty variables
# Make sure the exit code for the last failed step (if any) in the pipeline will be pipeline's exit code
set -u -o pipefail
```

Reply:
I don't think the line is self explanatory or a trivial command, could be good to add comments